rfc/x/index.md

```diff
@@ -0,0 +1,307 @@
+encounter anatomical information. Second, there is ambiguity in whether `R` refers to
+"starting from the right" or "going to the right". Third, since NGFF axes are encoded
+separately, it is most natural to encode each axis direction separately.
+
+A free-form value in `anatomicalOrientation` or `long_name` fields was considered, but
+an consistently understood controlled vocabuary is desired.
+
+Defining only a default implicit orientation was considered, but this did not meet the
+needs of real-world acquisitions.
+
+## Prior art and references
+
+###
+
+- [ITK Coordinate Systems](https://www.slicer.org/wiki/Coordinate_systems#Anatomical_coordinate_system)
+- [Nifti Orientation](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Orientation%20Explained#:~:text=A%20valid%20NIfTI%20image%20can,way%20aro)
+- [Allen Institute Data Schema](https://github.com/AllenNeuralDynamics/aind-data-schema/blob/24d3899823557c7b45d1cdff844993dee3c63e9d/src/aind_data_schema/imaging/acquisition.py#L26)
+- [Get Your Brain Together Hackathon Notes](https://github.com/InsightSoftwareConsortium/GetYourBrainTogether/blob/main/HCK02_2023_Allen_Institute_Hybrid/BoFBreakouts/Anatomic_orientation_in_OME-Zarr_NGFF_BoF_notes.md)
+- [3D Slicer Coordinate Systems documentation](https://slicer.readthedocs.io/en/latest/user_guide/coordinate_systems.html)
+- [Human Brain Project openMINDS anatomicalAxesOrientation](https://raw.githubusercontent.com/HumanBrainProject/openMINDS_controlledTerms/v1/instances/anatomicalAxesOrientation/RAS.jsonld)
+- [DICOM Image Position and Image Orientation C7.6.2.1.1](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.2.html#sect_C.7.6.2.1.1)
+
+Related and complementary is the [Nifti-Zarr draft standard](https://github.com/neuroscales/nifti-zarr/tree/draft/0.2), whose goal is to enhance OME-Zarr with Nifti metadata.
+However, this only implicitly allows an `RAS` orientation whereas this proposal enables explicit encoding with other orientation options.
+
+For simplicity and to address the most common use cases, only instances where a single organism are present in the image are support.
+
+## Future possibilities
+
+Future work may include expanding the controlled vocabulary based on community feedback and evolving requirements.
+
+## Performance
+
+Explicitly defining anatomical orientation is expected to improve the accuracy and efficiency of downstream analysis, but specific performance metrics will need to be defined and evaluated.
+
+## Backwards Compatibility
+
+The proposed change is backwards compatible, as it introduces a new optional field without altering existing fields.
+
+## Testing
+
+Testing will include validating the presence and correctness of the `anatomicalOrientation` field in the metadata and ensuring compatibility with existing tools and workflows.
+
+## UI/UX
+
+End-users applications SHOULD display the encoded information.
```

Comment:
Would be nice to give help on how to do it. One option (at least for DICOM images) is to refer to the [DICOM standard](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.html#sect_C.7.6.1.1.1). It would be a bit nicer to provide a yml or json file that describes these for each `anatomicalOrientation` enum:

```
{
    "positiveDirectionLabel", "left",
    "positiveDirectionAbbreviation": "L",
    "positiveDirectionCode": ["SCT", " 7771000", "Left"],
    "negativeDirectionLabel": "right",
    "negativeDirectionAbbreviation": "R",
    "negativeDirectionCode": ["SCT", "24028007", "Right"]
}
```

`...DirectionCode` is a coded term in some standard terminology. It could be used for translation to different languages. It is just a suggestion, if people find it too complex/unnecessary then it may be removed.

Reply:
Here I was not referring to code but rendering in applications -- I added more details.